ilp-core/src/main/java/org/interledger/core/InterledgerFulfillment.java

```diff
@@ -26,99 +26,103 @@
 import java.util.Objects;
 
 /**
- * Represents an Interledger Fulfillment (as present in an ILP Fulfill packet).
+ * <p>Represents an Interledger Fulfillment that can be used in an {@link
+ * InterledgerFulfillPacket}.</p>
  *
- * <p>This is a wrapper around the 32 byte pre-image from a SHA-256 hash digest.
+ * <p>This is a wrapper around the 32 byte pre-image of a SHA-256 hash digest.</p>
  *
  * @see InterledgerCondition
  */
 public interface InterledgerFulfillment extends Comparable<InterledgerFulfillment> {
 
   /**
-   * Create a new immutable InterledgerFulfillment from the provided 32-bytes.
+   * Create a new immutable InterledgerFulfillment from the provided 32-hash.
```

Comment:
`32-hash`

Reply:
It's a pre-image, not a hash right?

Reply:
Yep. Looks like a find-replace mistake.

Reply:
Fixed by c387f11431c2f1904d8b5bf3729568e3f95959b6